src/World.cc

```diff
@@ -900,6 +943,16 @@ sdf::ElementPtr World::ToElement(const OutputConfig &_config) const
         this->dataPtr->sphericalCoordinates->ElevationReference());
     sphericalElem->GetElement("heading_deg")->Set(
         this->dataPtr->sphericalCoordinates->HeadingOffset().Degree());
+    if (sphericalElem->HasElement("surface_axis_equatorial"))
+    {
+      sphericalElem->GetElement("surface_axis_equatorial")->Set(
+          this->dataPtr->sphericalCoordinates->SurfaceAxisEquatorial());
+    }
+    if (sphericalElem->HasElement("surface_axis_polar"))
```

Comment:
I would remove this `HasElement` check to maintain consistency with the other `ToElement` functions, and then add corresponding expectations to the `ToElement` test

Reply:
Done https://github.com/gazebosim/sdformat/pull/1050/commits/a74dd1068eb903f33a7c6611051380cb36cb8d2a